w3-ucan-bridge.md

```diff
@@ -0,0 +1,174 @@
+Content-Type: application/json
+
+{
+  tasks: [
+    ["store/add", "did:key:z6Mkm5qHN9g9NQSGbBfL7iGp9sexdssioT4CzyVap9ATqGqX", {
+        "link": "bagbaierah5sr5zt3tqgkrixptqzyerpxp5vwyjlx3n5frp2tbnr3clqrmrqa",
+        "size": 42
+    }],
+    ["store/add", "did:key:z6Mkm5qHN9g9NQSGbBfL7iGp9sexdssioT4CzyVap9ATqGqX", {
+        "link": "bafybeicajpuoxboivzka7cyft7okjf6vp43uk5udnedsrle6jews2cqj3a",
+        "size": 789
+    }]
+  ]
+}
+```
+
+And receive a DAG-JSON-encoded list of UCAN bridge receipts like:
+
+```json
+[
+  {
+    "data": {
+      "fx": {
+        "fork": []
+      },
+      "iss": "did:web:staging.web3.storage",
+      "meta": {},
+      "out": {
+        "ok": {
+          "before": "bafybeiabommx77q4ltcolzsmyqykuk6tsnerkixr6lsoegrx7qejcfurhu",
+          "results": [
+            {
+              "insertedAt": "2024-02-09T00:53:54.545Z",
+              "root": {
+                "/": "bafybeiabommx77q4ltcolzsmyqykuk6tsnerkixr6lsoegrx7qejcfurhu"
+              },
+              "shards": [
+                {
+                  "/": "bagbaierah5sr5zt3tqgkrixptqzyerpxp5vwyjlx3n5frp2tbnr3clqrmrqa"
+                }
+              ],
+              "updatedAt": "2024-02-09T00:53:54.545Z"
+            },
+            {
+              "insertedAt": "2024-02-16T05:11:35.173Z",
+              "root": {
+                "/": "bafybeicajpuoxboivzka7cyft7okjf6vp43uk5udnedsrle6jews2cqj3a"
+              },
+              "shards": [
+                {
+                  "/": "bagbaierahw552ajjkkxsvfgu5amm3o5bpzfhvrrl5vouwubwwk5wpbjiu5eq"
+                }
+              ],
+              "updatedAt": "2024-02-16T05:11:35.173Z"
+            }
+          ],
+          "size": 2
+        }
+      },
+      "prf": [],
+      "ran": {
+        "/": "bafyreienff66mf7nse3rm2njikxlshja6d5dkr4fpnyzeik5l6vils6fvq"
+      }
+    },
+    "sig": {
+      "/": {
+        "bytes": "7aEDQCiJYmYe9Gf25hv84NVN/fjN+udnT4Q65kVHFmQb1MPB2EmwDnR+S/TeYNkMxBdwIsNOKDwyTCqKOxvUhbWkGA8"
+      }
+    }
+  }
+]
+```
+
+A UCAN bridge receipt has two fields, `data` and `sig`:
+
+`data` contains the result of an invocation (in the nested `out` field) along with information about the invocation.
+`sig` is a signature over a CBOR-encoded version of the value of the `data` field.
+
+## Authorization
+
+The `X-Auth-Secret` and `Authorization` header values can be generated with the `bridge generate-tokens` command of `w3cli`:
```

Comment:
Before tuition on how to generate `X-Auth-Secret`, I'd like to know:

* What it is
* Why it is necessary
* How it is encoded